recipe/meta.yaml

```diff
@@ -52,7 +57,7 @@ outputs:
         - {{ pin_subpackage("libcudss" ~ somajor, max_pin="x.x.x") }}
     files:
       - lib/libcudss.so.*  # [linux]
-      - bin/cudss64_0.dll  # [win64]
+      - Library/bin/cudss64_0.dll  # [win]
```

Comment:
Will this work on Windows ARM?

Reply:
The difference between `win` and `win64` selectors is "any windows" vs "64-bit windows", so the real question is "Does this work on 32-bit Windows?" The answer is "No", but that's moot because conda-forge doesn't have separate 32-bit/64-bit Windows build variants.

Reply:
There are also no builds for cudss on Windows ARM, so we don't know what the filename would be for those builds.

Reply:
Historically that would have been true

In the near future `win64` will be used to distinguish Windows `x86_64` from Windows `arm64`: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5440

Hence the question about Windows ARM

It is ok that it is unsupported atm. Just thought we were trying to clarify between these newer Windows architectures

Reply:
Are they patching conda-build too? Because according to the docs, `win64` doesn't know about x86 vs arm.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors

Reply:
Conda already knows about it: https://github.com/conda/conda/pull/11778

Most likely we will need to compose OS & arch selectors using [this sort of approach]( https://github.com/conda-forge/go-activation-feedstock/blob/9d5e27360af59e7367500ae424e9786c74a8cf39/recipe/conda_build_config.yaml#L5-L6 )

For now `win64` has been used as shorthand for `win and x86_64`